build.gradle.kts

```diff
@@ -79,3 +80,10 @@ val formattedMessage =
   message.replace("\n====", "\n" + "=".repeat(message.lines().maxByOrNull { it.length }!!.length))
 
 logger.info(formattedMessage)
+
+apiValidation {
+  ignoredProjects += listOf("bench", "docs")
+  nonPublicMarkers += "org.pkl.commons.annotations.PklExperimental"
+
+  @OptIn(kotlinx.validation.ExperimentalBCVApi::class) klib { enabled = true }
+}
```

Comment:
It's best to configure this plugin in the root `build.gradle.kts`, because then projects can be ignored by name. I've enabled [experimental KLib validation](https://github.com/Kotlin/binary-compatibility-validator?tab=readme-ov-file#experimental-klib-abi-validation-support) since Pkl is well past the minimum of `1.9.20`.